src/form.js

```diff
@@ -0,0 +1,46 @@
+export default class EditForm {
+  constructor(taskList) {
+    this.taskList = taskList;
+    this.input = document.createElement('input');
+    this.input.type = 'text';
+    this.input.className = 'edit-input';
+    this.trash = document.createElement('button');
+    this.trash.type = 'button';
+    this.trash.innerHTML = '<i class="fa fa-trash"></i>';
+    this.trash.className = 'rightmost trash';
+  }
+
```

Comment:
- Kindly install the [correct linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) for your proyect.